src/Razor/test/testapps/Directory.Build.props

```diff
@@ -38,4 +38,26 @@
 
   <Import Project="After.Directory.Build.props" Condition="Exists('After.Directory.Build.props')" />
 
+  <!--
+    NOTE, BELOW
```

Comment:
@pranavkm This entire commit was the reason why the test apps weren't publishing.

Reply:
Why aren't these settings coming from /Directory.Build.targets and /eng/Versions.props -- at least indirectly? The Roslyn version there gets updated daily if there are new packages available.

Reply:
We don't have access to either of those when these apps are tested (they're published to a temp directory).

I could have written logic in the test project to harvest that info but it was a bit more complex than I felt was warranted given all of this is temporary. Once we get a preview 7 SDK I'll delete all of this.

Reply:
Please do not delete this logic or something that solves the problem without breaking automatic updates. The preview 7 SDK represents a temporary point of time when the Roslyn version aligns.

I'm wondering if this is a problem something in the AspNetCore repo has already solved in its test infrastructure. Probably worth asking around&hellip;